packages/inference/src/lib/makeRequestOptions.ts

```diff
@@ -56,13 +56,15 @@ export async function makeRequestOptions(
 		/** In most cases (unless we pass a endpointUrl) we know the task */
 		task?: InferenceTask;
 		chatCompletion?: boolean;
+		/* Used internally to generate inference snippets (in which case model mapping is done separately) */
+		__skipModelIdResolution?: boolean;
```

Comment:
```suggestion
		skipModelIdResolution?: boolean;
```

i would name it like that bc `__` is just too ugly:)

Reply:
done in https://github.com/huggingface/huggingface.js/pull/1255/commits/a280d60f855508cebfb0b43cdfe80e7972777146
Note that in the end this PR doesn't use this argument but will do as soon as I start the next PR (to fix openai/requests snippets for external providers)

Reply:
yes, that's what i figured